bin/updatestrings.sh

```diff
@@ -1,7 +1,20 @@
 #!/usr/bin/env sh
+# Extract string for default locale
 ../../../app/console translation:extract en -v \
   --dir=. \
   --exclude-dir=vendor --exclude-dir=Tests --exclude-dir=Features --exclude-dir=node_modules --exclude-dir=Resources/public/vendors \
   --output-dir=./Resources/translations \
   --keep
   "$@"
+
+# Clean corresponding ezplatform-i18n folder
+rm ../../ezsystems/ezplatform-i18n/source/platform-ui-bundle/*.xlf
```

Comment:
Maybe we should output something to say what is being done. The extraction script will output stuff, but the system commands below won't.